client/extensions/woocommerce/app/dashboard/manage-orders-view.js

```diff
@@ -53,21 +101,25 @@ class ManageOrdersView extends Component {
 	}
 
 	render = () => {
-		const { site, translate } = this.props;
-		// TODO - replace store owner with current user's first name
+		const { site, translate, orders, user } = this.props;
 		return (
 			<div className="dashboard__manage-has-orders">
 				<div className="dashboard__manage-has-orders-header">
 					<h2>
-						{ translate( 'Welcome back, store owner' ) }
-						<br />
-						{ translate( 'You have new orders to process' ) }
+						{ translate( 'Welcome back, {{storeOwnerName/}}.', {
+							components: {
+								storeOwnerName: <strong>{ user.display_name || user.username }</strong>
+							}
+						} ) }
+						{ orders.length && (
+							<span>{ translate( 'You have new orders to process' ) }</span>
+						) || '' }
 					</h2>
 				</div>
-				{ this.renderProcessOrdersWidget() }
+				{ this.possiblyrenderProcessOrdersWidget() }
 				<div className="dashboard__manage-has-orders-stats-actions">
 					<Button href={ getLink( '/store/stats/orders/day/:site', site ) }>
-						{ translate( 'View full reports' ) }
+						{ orders.length ? translate( 'View full reports' ) : translate( 'View reports' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'View Report' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).